src/Lib.hs

```diff
@@ -231,6 +217,12 @@ getAttachmentsFromComment comments = do
     isAttachmentZip :: Attachment -> Bool
     isAttachmentZip attachment = "application/zip" == aContentType attachment
 
+-- | Get zendesk responses
+getZendeskResponses :: [Comment] -> [Attachment] -> TicketInfo -> App [ZendeskResponse]
```

Comment:
Isn't the mapping one `TicketInfo` - one `ZendeskResponse`? We should fix this, as noted above.

Reply:
My first impression is that fixing this could be a little tricky. I'll try to fix it today and if I couldn't get it done by today, then I'll have to ask for your help